README.rst

```diff
@@ -55,6 +55,38 @@ major or minor **GitHub** release are designed to help developers to migrate fro
 one version to another. The changes listed are the steps you need to take to
 move from the previous version to the one listed.
 
+Installing Libdrizzle Redux
+===========================
+
+Debian
+------
+
+For ``Debian`` based systems running **ubuntu xenial** ``deb`` packages are
+available in the project's **apt** repository at `bintray`_.
+
+To add the **apt** repository to your system follow the instructions given
+when clicking on the **[set up me]!** button.
+
+.. image:: docs/_static/bintray-setmeup.png
```

Comment:
Y the underscore ?

Reply:
> Y the underscore ?

What do you mean by?

Reply:
`_static` ?

Reply:
just the name of the directory. Legacy naming from when we forked